Watchman.Discord/Program.cs

```diff
@@ -17,7 +20,36 @@ public static async Task Main(string[] args)
             var configuration = JsonConvert.DeserializeObject<DiscordConfiguration>(File.ReadAllText(configPath));
             
             var watchman = new WatchmanBot(configuration);
-            await watchman.GetWorkflowBuilder().Run();
+            var workflowBuilder = watchman.GetWorkflowBuilder();
+
+            var lastException = DateTime.MinValue;
+            var numberOfExceptionsInLastTime = 0;
+
+            while (true)
+            {
+                try
+                {
+                    await workflowBuilder.Run();
+                }
+                catch (Exception e)
+                {
+                    Log.Error(e, "Not handled exception!");
+                    if (numberOfExceptionsInLastTime > HOW_MANY_EXCEPTIONS_IN_SHORT_TIME_TO_STOP_BOT)
```

Comment:
```csharp
                    if (numberOfExceptionsInLastTime++ > HOW_MANY_EXCEPTIONS_IN_SHORT_TIME_TO_STOP_BOT)
                    {
                        break;
                    }
                    if (lastException < DateTime.Now.AddMinutes(-30))
                    {
                        numberOfExceptionsInLastTime = 0;
                    }
```

Reply:
często zapomina sie o tym że ++ jak jest na końcu, to dodaje +1 ale dopiero później